db/models.py

```diff
@@ -1,10 +1,12 @@
 from django.db import models
 
+from django.forms import CharField
+
 
 class Genre(models.Model):
     name = models.CharField(max_length=255, unique=True)
 
-    def __str__(self) -> str:
+    def __str__(self) -> CharField:
```

Comment:
The return type for the `__str__` method should be `str`, not `CharField`. Update the type annotation to `-> str`.